newIDE/app/src/Profile/ContributionsDetails.js

```diff
@@ -0,0 +1,234 @@
+            <Trans>Error retrieving the extensions</Trans>
+          </Text>
+        </Line>
+      </Column>
+    );
+
+  return extensions ? (
+    <Accordion>
+      <AccordionHeader>
+        <Text displayInlineAsSpan>
+          <Trans>{`Extensions (${extensions.length})`}</Trans>
+        </Text>
+      </AccordionHeader>
+      <AccordionBody disableGutters>
+        <Column>
+          {extensions.map(extension => (
+            <ContributionLine
+              shortDescription={extension.shortDescription}
+              fullName={extension.fullName}
+              previewIconUrl={extension.previewIconUrl}
+            />
+          ))}
+        </Column>
+      </AccordionBody>
+    </Accordion>
+  ) : (
+    <PlaceholderLoader />
+  );
+};
+
+// Change examples to assets when the feature is developed.
+type AssetsAccordionProps = {|
+  examples: ?Array<ExampleShortHeader>,
+|};
+
+const AssetsAccordion = ({ examples }: AssetsAccordionProps) => {
+  return examples ? (
+    <Accordion>
+      <AccordionHeader>
+        <Text displayInlineAsSpan>
+          <Trans>Assets (coming soon!)</Trans>
+        </Text>
+      </AccordionHeader>
+    </Accordion>
+  ) : (
+    <PlaceholderLoader />
+  );
+};
+
+type Props = {|
+  userId: string,
+|};
+
+export default ({ userId }: Props) => {
+  const [
+    extensions,
+    setExtensions,
+  ] = React.useState<?Array<ExtensionShortHeader>>(null);
+  const [examples, setExamples] = React.useState<?Array<ExampleShortHeader>>(
+    null
+  );
+  const [extensionError, setExtensionError] = React.useState<?Error>(null);
+  const [exampleError, setExampleError] = React.useState<?Error>(null);
+
+  const fetchUserExtensions = React.useCallback(
+    () => {
+      (async () => {
+        if (!userId) return;
+        try {
+          const extensions = await getUserExtensionShortHeaders(userId);
+          setExtensions(extensions);
+        } catch (error) {
+          console.error('Error while loading extensions:', error);
+          setExtensionError(error);
+        }
+      })();
+    },
+    [userId]
+  );
+
+  React.useEffect(
+    () => {
+      fetchUserExtensions();
+    },
+    [fetchUserExtensions]
+  );
+
+  const fetchUserExamples = React.useCallback(
+    () => {
+      (async () => {
+        if (!userId) return;
+        try {
+          const examples = await getUserExampleShortHeaders(userId);
+          setExamples(examples);
+        } catch (error) {
+          console.error('Error while loading examples:', error);
+          setExampleError(error);
+        }
+      })();
+    },
+    [userId]
+  );
+
+  React.useEffect(
+    () => {
+      fetchUserExamples();
+    },
+    [fetchUserExamples]
+  );
+
+  return (
+    <>
+      {extensions && examples && (
+        <Column>
+          <Line alignItems="center">
+            <Text size="title">
+              <Trans>Contributions</Trans>
+            </Text>
+          </Line>
+        </Column>
+      )}
```

Comment:
I think we can this title always shown (it's part of the structure of the page I would say).